packages/is-prop-valid/src/props.js

```diff
@@ -1,3 +1,6 @@
+/**
+ * This module needs to remain pure JavaScript for codegen to work on it
```

Comment:
this makes sense 👍 making this go through something like `ts-node` would be overengineered, I'm OK with this being a JS file